engine/src/mbldc.cpp

```diff
@@ -241,6 +241,11 @@ void MCScreenDC::handle_motion(MCEventMotionType p_type, double p_timestamp)
 
 void MCScreenDC::process_touch(MCEventTouchPhase p_phase, void *p_touch_handle, int32_t p_timestamp, int32_t p_x, int32_t p_y)
 {
+    if (m_current_window == nil)
```

Comment:
It would be worth adding a comment here:
```
/* It is possible for the engine to receive touch messages between initial startup and
 * there being a stack shown, so ignore the touch if there is no current window. */

Reply:
ok